api/bazel/repositories.bzl

```diff
@@ -31,6 +35,16 @@ def api_dependencies():
         strip_prefix = "protoc-gen-validate-" + PGV_RELEASE,
         url = "https://github.com/lyft/protoc-gen-validate/archive/v" + PGV_RELEASE + ".tar.gz",
     )
+    http_file(
```

Comment:
will be replaced with http_archive or something similar to download kafka 2.3 client jar, then it will be unpacked to get all kafka request `.json` spec files